src/cpu/CMakeLists.txt

```
@@ -38,6 +38,14 @@ if((DNNL_TARGET_ARCH STREQUAL "X64") OR (DNNL_TARGET_ARCH STREQUAL "AARCH64"))
     endforeach()
 endif()
 
+if(DNNL_TARGET_ARCH STREQUAL "PPC64")
+    file(GLOB FILES_REQUIRED_OPT
+        ${CMAKE_CURRENT_SOURCE_DIR}/gemm/*.[ch]pp
+    )
+    set_source_files_properties(${FILES_REQUIRED_OPT}
+        PROPERTIES COMPILE_FLAGS "-O3 -funroll-loops")
```

```suggestion
    if(NOT UPPERCASE_CMAKE_BUILD_TYPE STREQUAL "DEBUG")
        set_source_files_properties(${FILES_REQUIRED_OPT}
            PROPERTIES COMPILE_FLAGS "-O3 -funroll-loops")
    endif()
```

Maybe guard this so that optimizations are not applied in debug build?